ci-operator/step-registry/openshift-qe/cluster-density/openshift-qe-cluster-density-commands.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+oc config view
+oc projects
+python3 -m pip install virtualenv --user
```

Comment:
Perhaps we can move it into the Dockerfile/image, so we do not need to repeat the install every time we test, which saves us time/resource.

Reply:
Yep. Moving it there !!